### ISSUE TITLE
Short description is only visible before first mod is selected

### ISSUE THREAD

**Issue by schorsch3000** [6thmoon/OptionGenerator#2]
When the mod tab is selected in settings and the right big pane is empty hovering a mod shows a short description of that mod.

Once any mod is selected there is no way to get back to the description bit selecting any other setting than mod setting and going back to mod settings.

I think it would be nice if there would be a back button to "leave" the settings of a mod and go back to the empty pane that shows the description

**Comment by 6thmoon** [comment]
Yep, this is just how **Risk of Options** works. Still might add it since I would like this change as well.
